modules/API/pages/built-in-endpoints.adoc

```diff
@@ -9,22 +9,22 @@ This page describes the REST API endpoints accessible on a TigerGraph server.
 ====
 *Note for users of TigerGraph Cloud*
 
-For all TigerGraph Cloud solutions created on or after June 20, 2022, ports 14240 and 9000 are no longer exposed.
+For all TigerGraph Cloud solutions created on or after June 20, 2022, port 14240 are no longer exposed.
 This also applies to previously created cloud solutions that are resized on or after June 20, 2022. Use port 443 instead for API endpoint connectivity on cloud solutions:
 
-** Use API endpoint `443/restpp/<endpoint>` for RESTPP endpoints that were on port 9000
-
 ** Use API endpoint `443/<endpoint>` for endpoints that were on port 14240
 
-For all TigerGraph Server solutions not on TigerGraph Cloud, or for TigerGraph Cloud solutions created before June 20, 2022, ports 9000 and 14240 remain correct.
+For TigerGraph Cloud solutions created before June 20, 2022, ports 9000 and 14240 remain correct.
+
+In 4.1.0 and later versions, for all TigerGraph Server solutions not on TigerGraph Cloud, port 9000 are no longer exposed, please use `14240/restpp/v1/<endpoint>` instead.
```

Comment:
"/restpp/v1" -> "/restpp"
after 4.1.1, we will add version

Reply:
Done